src/HasTranslations.php

```diff
@@ -2,6 +2,7 @@
 
 namespace Spatie\Translatable;
 
+use Locale;
```

Comment:
Is there a full facade class available for this?

Reply:
Hm, not 100% sure if I got you. Shall I create a `Locale` class that extends `Facade`?
```php
<?php

namespace Spatie\Translatable;

use Illuminate\Support\Facades\Facade;

class Locale extends Facade
{
    protected static function getFacadeAccessor()
    {
        return 'locale';
    }
}
```

And then register a default implementation in `TranslatableServiceProvider`?

Reply:
O, I thought `Locale` was some kind of Laravel Facade, turns out it is a native PHP class.

Remind me again why we need to use this? Can't we trust an array where the keys are strings to be valid locales?

Reply:
That would not work with JSON fields in case you pass a JSON object (i.e a PHP array).

Reply:
I'm thinking validating this might be overkill, let's just assume the user did pass an array with locales.

Reply:
Okay, it's implemented but now it's a breaking change...